ci/Dockerfile

```
@@ -26,7 +26,7 @@ RUN poetry install --no-root --no-ansi $POETRY_ARGS && rm -rf $POETRY_CACHE_DIR
 ########################################################################################################################
 # python 3.12 is supported until Oct 2028
 # alpine 3.21 is supported until 1 Nov 2026
-FROM python:3.12-alpine3.21
+FROM python:3.13.2-alpine3.21
```

Why pin the patch version?  We weren't doing that with 3.12.
```suggestion
FROM python:3.13-alpine3.21
```

---

Sure, I begrudgingly pinned that patch, for some reason thinking it was done like that before.